app/Models/Manufacturer.php

```diff
@@ -23,8 +23,8 @@ class Manufacturer extends SnipeModel
     protected $rules = [
         'name'   => 'required|min:2|max:255|unique:manufacturers,name,NULL,id,deleted_at,NULL',
         'url'   => 'url|nullable',
-        'support_url'   => 'url|nullable',
         'support_email'   => 'email|nullable',
+        'support_url'   => ['starts_with:http://,https://,afp://,facetime://,file://,irc://','nullable'],
```

Comment:
Do we need to use brackets here? I don't think we do - just 

` 'support_url'   => 'nullable|starts_with:http://,https://,afp://,facetime://,file://,irc://',` 

should do it, no? I'd prefer that for consistency if it works.

Reply:
all set